### PR TITLE
#1169 ConditionalJsonResources supports Cache-Control: no-cache header.

### PR DESCRIPTION
**Pull request by criske**
PR for #1169

### CODE REVIEW

**Comment by zoeself** [comment]
@criske thank you for your Pull Request. I'll assign someone to review it soon.

File: self-core-impl/src/main/java/com/selfxdsd/core/GithubPersonalRepos.java [hunk 1]
```diff
@@ -83,7 +83,7 @@
     }
 
     /**
-     * @todo #1166:60min Possible bug in class FromHeaders. The bellow method
+     * TODO #1166:60min Possible bug in class FromHeaders. The bellow method
```
Comment (criske) [inline self-core-impl/src/main/java/com/selfxdsd/core/GithubPersonalRepos.java:86]: @amihaiemil here checkstyle complained about not starting the sentence with a letter.

**Comment by zoeself** [comment]
@amihaiemil I couldn't find any assignee for this task. This is either because there are no contributors with role ``REV`` available or because the project does not have enough funds.

Please, make sure there is at least one available contributor with the required role and the project can afford to pay them.

**Comment by amihaiemil** [comment]
@criske mersi, fain :D 

Apropo, cu TODO-u ala avem un bug [interesant](https://github.com/self-xdsd/self-core/issues/1171). Desteptii astia de la Oracle au specificat ca ``HttpHeaders.map()`` sa nu faca split dupa virgula.

Daca are primeste ``header: value1, value2, value3`` el va adauga in Map<String, List<String>> o Lista cu un singur element String ``"value1, value2, value3"`` (asta ii raspuns misto la interviu daca te intreaba de bug-uri in JDK...)

**Comment by amihaiemil** [comment]
@rultor merge

**Comment by criske** [comment]
> @criske mersi, fain :D 
> 
> Apropo, cu TODO-u ala avem un bug [interesant](https://github.com/self-xdsd/self-core/issues/1171). Desteptii astia de la Oracle au specificat ca ``HttpHeaders.map()`` sa nu faca split dupa virgula.
> 
> Daca are primeste ``header: value1, value2, value3`` el va adauga in Map<String, List<String>> o Lista cu un singur element String ``"value1, value2, value3"`` (asta ii raspuns misto la interviu daca te intreaba de bug-uri in JDK...)

@amihaiemil n-am fost atent la todo :)) nu-mi vine sa cred... Noaptea mintii! se pare ca nu e "bug it's a feature" _In particular, HttpHeaders does not perform any splitting or joining of comma separated header value strings_

**Pull request merged**